src/whatsapp/collections/LabelCollection.ts

```diff
@@ -29,7 +29,7 @@ export declare class LabelCollection extends Collection<LabelModel> {
   removeAllLabelsMD(chat: ChatModel): any;
   addOrRemoveLabelsMD(e?: any, t?: any): any;
   addOrRemoveLabels(e?: any, t?: any): any;
-  deleteLabel(e?: any): any;
+  deleteLabel(e?: string): any;
```

Comment:
Como que para deletar uma label é obrigatório passar o ID, então pode se remover a interrogação, seria bom também renomear para ficar mais claro
```suggestion
  deleteLabel(id: string): any;
```